project/templates/articles/details.html

```diff
@@ -33,43 +33,58 @@
 </script>
 
 <div class="article-info">
-    <h1 class="article-title">{{title}}</h1>
+    <h1>{{title}}</h1>
+    <h4>Subtitle: Lorem ipsuummmmmmm</h4>
```

Comment:
Is it a placeholder? Will we have to add this column attribute to Article's class?

Reply:
Yes, it is! I should have added a comment. -.-'

Reply:
Don't worry it's okey. We should create a task for add this attribute